### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by jonsequitur**

### CODE REVIEW

File: docs/Features-overview.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ ANSI terminals support a variety of features by including ANSI escape sequences
 
 The following are examples of output rendered by the same view code in these three different contexts.
 
-In PowerShell on Windows with ANSI mode enabled:
+In PowerShell on Windows with VT mode enabled:
```
Comment (loic-sharma) [inline docs/Features-overview.md:92]: What is VT? Consider expanding the acronym the first time you refer to "VT". Also, consider adding a link to something that may help introduce VT to CLI newbies :)

File: docs/model-binding.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+So for example, you might want an input `123` from the command line to be converted into an `int` with the value `123`. This conversion of command line input into variables or arguments that you can use in your code is called "binding." The term "model binding" refers to binding simple types as well as more complex types in order to pass the values to a method.
+
+# Binding parameters to a command handler
+
+The simplest way to bind command line input is to set the `Handler` property on a `Command`. The `System.CommandLine` model binder will look at the options and arguments for the command and attempt to match them to the parameters of the specified handler method. The default convention is that parameters are matched by name, so in the following example, option `--an-int` matches the parameter named `anInt`. Matching ignores hyphens (and other option prefixes, such as `'/'`) and is case insensitive.
+
+``` cs --source-file ./src/Binding/HandlerBindingSample.cs --project ./src/Binding/Binding.csproj --region MultipleArgs --session MultipleArgs
+var command = new RootCommand
+            {
```
Comment (loic-sharma) [inline docs/model-binding.md:29]: There appears to be a little too much indentation

Comment (loic-sharma) [reply docs/model-binding.md:29]: Same comment for lines 56-58

File: docs/Syntax-Concepts-and-Parser.md [hunk 3]
```diff
@@ -76,27 +76,11 @@ An argument is a value passed to an option or command.
...
-|      |      |                         | --file a.json b.json           |
-| 0    | _n_  | Valid:                  | --file                         |
-|      |      |                         | --file a.json                  |
-|      |      |                         | --file a.json b.json           |
-| 1    | _n_  | Valid:                  | --file a.json                  |
-|      |      |                         | --file a.json b.json           |
-|      |      | Invalid:                | --file                         |
-
-### Bundling
+MinMaxExamples01Valid:--flag--flag true--flag falseInvalid:--flag false false11Valid:--file a.jsonInvalid:--file--file a.json b.json0_n_Valid:--file--file a.json--file a.json b.json1_n_Valid:--file a.json--file a.json b.jsonInvalid:--file### Bundling
```
Comment (jonsequitur) [inline docs/Syntax-Concepts-and-Parser.md:83]: This looks like a `dotnet try publish` bug.

File: docs/model-binding.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                new Option<string>("--a-string"),
+                new Option<int>("--an-int")
+            };
+
+command.Handler = CommandHandler.Create(
+    (string aString, int anInt) =>
+    {
+        Console.WriteLine($"{aString}");
+        Console.WriteLine($"{anInt}");
```
Comment (loic-sharma) [inline docs/model-binding.md:38]: What do you think of this?

```suggestion
        Console.WriteLine(aString);
        Console.WriteLine(anInt);
```

Comment (jonsequitur) [reply docs/model-binding.md:38]: Amazing!

**Pull request merged**